stdlib/LinearAlgebra/test/generic.jl

```diff
@@ -248,6 +253,18 @@ end
     end
 end
 
+@testset "normalize for multidimensional arrays" begin
+
+    for arr in ([1.0 2.0 3.0; 4.0 5.0 6.0], OffsetArray([-1,0], (-2,)))
```

Comment:
Adding a few higher dimensional arrays can't hurt:
```suggestion
    for arr in ([1.0 2.0 3.0; 4.0 5.0 6.0], OffsetArray([-1,0], (-2,)), rand(1,2,3), rand(1,2,3,4))
```

Reply:
Okay this is super nitpicky, but we should also test the zero dimensional case. You can create zero dimensional arrays like:
```julia
julia> fill(10.0, ())
0-dimensional Array{Float64,0}:
10.0
```

Reply:
ok! with ae84cb3 there are now Array{Float64,0}, Array{Float64,1}, Array{Float64,3}, Array{Float64,3}, Array{Float64,4} test cases as well as the offset array and the Int64 case.

Reply:
Thanks again @ssikdar1 for doing this and all your patience for pedantic requests.

Reply:
yup! no worries!